docs/Reference/API-Methods.md

```diff
@@ -99,68 +169,107 @@ To return the Privacy Group ID with the payload, use the `receive` method with t
      }
     ```
 
-## sendraw
+## receiveraw
 
-Sends a raw payload to Orion.
+Receives a raw payload from Orion using the payload key. The payload key is returned by the [sendraw](#sendraw) method
 
 **HTTP Verb**
 POST 
 
 **Headers:**
 Content-Type: application/octet-stream
-c11n-from: Public key of the sender
-c11n-to: List of public keys to receive this payload
+c11n-key: Key used to receive the payload
 
 **Request Body**
 
-`payload` : *string* - Payload
+None
 
 **Returns**
 
-Key used to receive the payload
+Payload
 
 !!! example
     ```bash tab="curl HTTP request"
-    curl -X POST http://127.0.0.1:8888/sendraw \
+    curl -X POST \
+      http://127.0.0.1:8888/receiveraw \
       -H 'Content-Type: application/octet-stream' \
-      -H 'c11n-from: 4xanJzyaDPcBVMUSwl/tLp+DbXzd3jF9MKk1yJuyewE=' \
-      -H 'c11n-to: YE5cJRJYTRO4XFo7yuAi/0K9DwjySGjsHB2YrFPnJXo=' \
-      -d 'Hello, World!'
+      -H 'c11n-key: +3gnwO0oHXe4kXsr3kegd9jTTqsq3Y6Hm3w26WHR/RM='
     ```
    
     ```json tab="Result"
-    +3gnwO0oHXe4kXsr3kegd9jTTqsq3Y6Hm3w26WHR/RM=
+    Hello, World!
     ```
 
-## receiveraw
+## send
 
-Receives a raw payload from Orion using the payload key. The payload key is returned by the [sendraw](#sendraw) method
+Sends a payload to Orion.
```

Comment:
We can also `send` using `privacyGroupId`. A sample request is as follows 
```
curl -X POST \
  http://127.0.0.1:9002/send \
  -H 'Content-Type: application/json' \
  -d '{
    "payload": "SGVsbG8sIFdvcmxkIQ==",
    "from": "A1aVtMxLCUHmBVHXoZzzBgPbW/wj5axDpW9X8l91SGo=",
    "privacyGroupId": "DyAOiF/ynpc+JXa2YAGB0bCitSlOMNm+ShmB/7M6C4zgmFoVyvJ5qxREuOQw5IQhe3IzeA=="
}'
```
The output from the request is same as that of `send`.

Reply:
Fixed